app/modules/core/module.go

```diff
@@ -13,7 +18,7 @@ func (m *Module) GetName() string {
 // GetCommands returns the commands provided by this module.
 func (m *Module) GetCommands() []interfaces.Command {
 	return []interfaces.Command{
-		&LookupCommand{},
+		&LookupCommand{m.Config},
```

Comment:
is there an easier way to get the config here than to drag it through with the module?

Reply:
yeah i can create a new param or struct which can take care of these overrides instead of passing the config object all the way to the command but if that requires changing too many things then its best to just fallback to passing the config to the command.

i will update the patch this weekend.

thanks for reviewing the changes